yt_dlp/extractor/mediaset.py

```diff
@@ -218,6 +294,29 @@ def _real_extract(self, url):
         return info
 
 
+class MediasetClipIE(MediasetIE):
+    _VALID_URL = r'''(?x)
+                     https?://
+                         (?:www\.)?\w+\.mediaset\.it/
+                             video/[\w-]+_(?P<id>\d{5,})\.s?html?
+                 '''
+
+    _TESTS = [{
+        'url': 'https://www.grandefratello.mediaset.it/video/ventinovesima-puntata_27071.shtml',
+        'info_dict': {
+            'id': 'F310293901002901',
+            'ext': 'mp4',
+        },
+        'skip': 'DRM content only',
+    }]
```

Comment:
Is there no non-DRM test?

Reply:
nope, everything is DRM.

Now that I think about it, should we just remove the class, as it is useless to download. It only serves the purpose ot relling people that the content is DRM protected so they don't open an Issue on Github to support the website.

Reply:
yeah, we shouldn't add a extractor for just DRM content

Reply:
So far it was just the test URL that someone asked about. But I don't have access to Italian TV so I have no idea what other content might match the `_VALID_URL`.

Reply:
ok, reverted the related commit.